### ISSUE TITLE
fopen error on long cacheID

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/minify#299]
```
Minify commit/version: 2.1.7
PHP version: 5.3.24

What steps will reproduce the problem?
1. Test on Apache web-server setup on Windows computer
2. Minify 20+ media files in the same Minify string

Expected output: proper minified contents.

Actual output: PHP error:

fopen(xxxx/xxxx/cache/minify_adminbar.min,functions,wordcount.min,quicktags.min,
jquery.ui.widget.min,jquery.ui.mouse.min,jquery.ui.resizable.min,jquery.ui.dragg
able.min,jquery.ui.button.min_002c133e7a41ae60417bec568cd77311) [<a 
href='function.fopen'>function.fopen</a>]: failed to open stream: Invalid 
argument in <b>xxxx\xxxx\min\lib\Minify\Cache\File.php</b> on line <b>100</b>

The cacheId seems to be too long on Windows computer.

Please advise, thank you.
```

---

Original issue reported on code.google.com by `mr.minhk...@gmail.com` on 27 Oct 2013 at 2:58

**Comment by GoogleCodeExporter** [comment]
```
See this line: 
http://code.google.com/p/minify/source/browse/min/lib/Minify.php#568

Does changing 200 to 100 fix it?
```

---

Original comment by `mrclay....@gmail.com` on 27 Oct 2013 at 3:59

**Comment by GoogleCodeExporter** [comment]
```
Yes, I did change 200 to 100 for the current version of my plugin and it seems 
to work. Not sure if such change affects anything or is the long name there 
just for info? The md5 is quite sufficient already.
```

---

Original comment by `mr.minhk...@gmail.com` on 27 Oct 2013 at 4:02

**Comment by GoogleCodeExporter** [comment]
```
The long filenames were to assist people in debugging Minify and deleting 
certain cache files manually. 100 should be a lot safer.
```

---

Original comment by `mrclay....@gmail.com` on 27 Oct 2013 at 4:10

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 6e0e130a5bfd.
```

---

Original comment by `mrclay....@gmail.com` on 27 Oct 2013 at 4:12
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision f8b62b358b53.
```

---

Original comment by `mrclay....@gmail.com` on 14 Nov 2013 at 1:41
